src/CharacterCard.js

```diff
@@ -1,7 +1,26 @@
 import React from 'react';
+// import QuoteButton from './QuoteButton'
+import quotes from './quotes.json';
 
 class CharacterCard extends React.Component {
-  render() {
+  constructor(){
+    super();
+
+    this.state = { display: "" }
+  }
+  
+  getQuote(name){
+    const getQuotes = quotes[name];
+    const numberOfQuotes = getQuotes.length
+    if(getQuotes !== undefined){
+      const num = Math.floor(Math.random() * numberOfQuotes);
+      const quote = getQuotes[num];
+      this.setState({ display: quote });
+    }
+  }
```

Comment:
so something you could do is use the double `!!`. Let's see if I can explain it here.

so let's say `getQuotes` returns an array of two quotes. If I then call `getQuotes`, it would return the array. Now if I added a `!` in front of that, it would actually return `false`. Since getQuotes returns an actual value, and since the exclamation mark is like a "not" -- "NOT getQuotes" would return the opposite of a returned value... false! 
![image](https://user-images.githubusercontent.com/21206175/84163322-5ee5c700-aa3f-11ea-9fb8-9ba3cb23b9bf.png)

Now.... what would happen if we added TWO exclamation marks? `!!getQuotes`. This basically does a double NOT. Now what would be the opposite of false? True! That would mean that `!!getQuotes` would return true.
![image](https://user-images.githubusercontent.com/21206175/84163712-cac82f80-aa3f-11ea-8911-ed1b3ef968bb.png)

So whats the point of this? Basically, if you ever want to check that an array, or an object has values attributed to it, you can simply check with the double !!. 

This means that `getQuotes !== undefined` is the same as `!!getQuotes`, except the latter is a lot more "elegant" and cleaner.

Reply:
TLDR;; 
It basically turns an array into a true or false statement, if you want to check that the array is not empty.